comp/volunteers/saturday-meal.yaml

```
@@ -12,3 +12,11 @@ description: >-
   We typically go out on the Saturday evening as volunteers to relax and
   share a meal together. This is normally organised centrally, though
   volunteers pay for their meal themselves.
+
+  Required actions:
+   - [ ] Collect attendance numbers for the meal
```

Collecting dietary restrictions at the same time helps a lot

```suggestion
   - [ ] Collect attendance numbers for the meal, along with dietary restrictions
```